### PR TITLE
Update tests accounts #36

### LINKED ISSUE #36: Update tests accounts
Increase account balances by multiplying on 1000

### PR DESCRIPTION
**Pull request by andy-sam**
Fixes #36

### CODE REVIEW

File: src/main/resources/db/changelog/changelog-2021-07-14-IncreaseAccountBalances.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+                  value: 500000
+            tableName: account
+            where: number = "2222222222222222"
+        - update:
+            columns:
+              - column:
+                  name: balance
+                  value: 150000
+            tableName: account
+            where: number = "3333333333333333"
```
Comment (alex-anakin) [inline src/main/resources/db/changelog/changelog-2021-07-14-IncreaseAccountBalances.yaml:26]: add empty line

File: src/main/resources/db/changelog/changelog-2021-07-14-IncreaseAccountBalances.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      id: update-account-balances
+      author: andy-sam
+      changes:
+        - update:
+            columns:
+              - column:
+                  name: balance
+                  value: 1000000
+            tableName: account
+            where: number = "1111111111111111"
```
Comment (alex-anakin) [inline src/main/resources/db/changelog/changelog-2021-07-14-IncreaseAccountBalances.yaml:12]: Despite account numbers are unique unless you add an index for it it's better to use the `where id=" clause for any select or update. Please fix all clauses.

**Review by alex-anakin — APPROVED** [review]
There are a couple of minor changes needed. Please fix, commit and then you can merge it.

File: src/main/resources/db/changelog/changelog-2021-07-14-IncreaseAccountBalances.yaml (hunk 1 above)
Comment (andy-sam) [reply src/main/resources/db/changelog/changelog-2021-07-14-IncreaseAccountBalances.yaml:26]: done

File: src/main/resources/db/changelog/changelog-2021-07-14-IncreaseAccountBalances.yaml (hunk 2 above)
Comment (andy-sam) [reply src/main/resources/db/changelog/changelog-2021-07-14-IncreaseAccountBalances.yaml:12]: done

**Pull request merged**